tests/Neo.VM.Tests/Helpers/EvaluationStackHelper.cs

```diff
@@ -0,0 +1,23 @@
+// Copyright (C) 2015-2024 The Neo Project.
+//
+// EvaluationStackHelper.cs file belongs to the neo project and is free
+// software distributed under the MIT software license, see the
+// accompanying file LICENSE in the main directory of the
+// repository or http://www.opensource.org/licenses/mit-license.php
+// for more details.
+//
+// Redistribution and use in source and binary forms with or without
+// modifications are permitted.
+
+using Neo.VM;
+using System.Linq;
+
+namespace Neo.Test.Helpers;
+
+public static class EvaluationStackHelper
+{
+    public static string Print(this EvaluationStack stack)
+    {
+        return $"[{string.Join(", ", stack.Select(p => $"{p.Type}{p}"))}]";
```

Comment:
In my opinion, we should move this to `ToString` no sense to duplicate code everywhere, and also is useful for debug

Reply:
so you still need to have one in VM test and one in devpack test, what is the difference.

Reply:
It doesn't hurt, and is good to override `ToString` to see something useful during debug, writes, etc

Reply:
There should a Interface or some way to hook into the VM for `override`. 

Please stop rushing code designs. It clear that no thought is put into the design when making code changes. Copy and Paste isn't the solution. I am not talking to anyone person, but us as a whole. I do it too. But we need to all stop and have plans. I think we should have in the 1st comment of the PR, what the plan is for design when doing design changes. Yes it takes more time, But we need the time to refactor and think it through.